subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_cell_pos

```diff
@@ -8,4 +8,5 @@ config NRF_CLOUD_CELL_POS
 	imply FPU
 	depends on MODEM_INFO
 	depends on MODEM_INFO_ADD_NETWORK
+	depends on NRF_CLOUD_MQTT
```

Comment:
We should have a discussion whether this should be `depends on` or `select`.
I would put `select` but cloud team should comment on that.

Reply:
Changed to select. If anyone objects to this change, speak now.

Reply:
@tejlmand Can you instruct us on whether to use select or depends on here? I'm tempted to use select here and in general in many place in order to avoid applications having to select something that should be selected anyway. I know this is against Kconfig documentation (https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt) and I understand the implications for menuconfig use case.
However, is this valid anymore in the Kconfig documentation:
`By abusing select you are able to select a symbol FOO even if FOO depends on BAR that is not set.`
When I try to repro this, I still get a Kconfig warning which makes sense.

Reply:
I think we could change this back to `depends on`. The reason I'm leaning towards `depends on` instead of `select` in this specific case is that selecting MQTT automatically behind the scenes changes the behavior of other features such as A-GPS and P-GPS too so user should really do that selection intentionally.

Reply:
Changed back to depends on as suggested. Shall we call it a day with this?

Reply:
Yep, sorry for changing back and forth.